<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Debuger</class>
 <widget class="QWidget" name="Debuger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <layout class="QVBoxLayout" name="Display">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>变量/表达式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="DisplayView">
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>值</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="DisplayLine">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DisplayButton">
           <property name="text">
            <string>添加变量/表达式</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="Buttons">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="Start">
           <property name="styleSheet">
            <string notr="true">font: 75 14pt &quot;Consolas&quot;;</string>
           </property>
           <property name="text">
            <string>开始调试</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Stop">
           <property name="styleSheet">
            <string notr="true">font: 75 14pt &quot;Consolas&quot;;</string>
           </property>
           <property name="text">
            <string>停止调试</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="StepInto">
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;Consolas&quot;;</string>
         </property>
         <property name="text">
          <string>StepInto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StepOut">
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;Consolas&quot;;</string>
         </property>
         <property name="text">
          <string>StepOut</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StepOver">
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;Consolas&quot;;</string>
         </property>
         <property name="text">
          <string>StepOver</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Continue">
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;Consolas&quot;;</string>
         </property>
         <property name="text">
          <string>Continue</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
